appvm-scripts/qubes-gui-agent.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Qubes GUI Agent
 After=qubes-meminfo-writer.service qubes-mount-dirs.service
+!ConditionPathExists=/var/run/qubes-service/lightdm
```

Comment:
Should be:
```suggestion
ConditionPathExists=!/var/run/qubes-service/lightdm
```

Reply:
Thanks!!! I wouldn't have guested this is the problem.